### ISSUE TITLE
Accessibility: Fix Screen Reader Functionality of FormItem; Remove title attribute from FormItemLabel

### ISSUE THREAD

**Issue by lgants** [ant-design/ant-design#31766]
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/antd-reproduction-template-forked-gjgf0?file=/index.js)

### Steps to reproduce
1. Open the reproduction link in Chrome
2. Enable MacOS VoiceOver utility and focus the "First Name" field - the form label is not read aloud
3. Inspect the Form.Item "First Name" label in your browser dev tools 
4. Delete the "title" attribute on the label element in dev tools
5. Focus the "First Name" field - the form label is read aloud

### What is expected?
The field label should be read aloud by MacOS VoiceOver utility when the field is focused 

### What is actually happening?
The field label is not read aloud by MacOS VoiceOver utility

| Environment | Info |
|---|---|
| antd | 4.16.11 |
| React | react |
| System | MacOS |
| Browser | Chrome |

---
The fix seems as simple as removing `title={typeof label === 'string' ? label : ''}` from the FormItemLabel.tsx component. But, I can't tell from the git history why the label element received the title attribute. I'm happy to create a PR, but I want to validate that attribute is not necessary for other use cases before creating a PR.

**Comment by afc163** [comment]
The field label can be read byMacOS VoiceOver in my machine.

**Comment by lgants** [comment]
@afc163 hmm interesting. I've checked versions again and I've got latest Chrome and latest MacOS. The same behavior also occurs on latest Safari. However, it does work correctly on latest Firefox. Do we know why the title attribute exists? If it doesn't server a purpose, it should be removed regardless right?

**Comment by lgants** [comment]
Also, just FYI my issue does include a reproduction link.

**Comment by afc163** [comment]
1. Your codesandbox doesn't include Form component.
2. Could you record a video to show how Screen Reader cannot work?

**Comment by lgants** [comment]
@afc163 sorry, my mistake. I must have forgotten to save the fork. A gif is below. Notice that when the input field is focused it should read the label aloud. Instead, the label is "blank". I'm experiencing this behavior on all ant form fields on Safari and Chrome.

Before (BAU):
![2021-08-12 00-23-17 2021-08-12 00_24_15](https://user-images.githubusercontent.com/18373573/129138076-71ba5a0f-4693-4915-b157-8d21bdc4d5de.gif)

After removing the title attribute from the label in dev tools:
![2021-08-12 00-30-37 2021-08-12 00_31_27](https://user-images.githubusercontent.com/18373573/129138611-0d9a9970-2b3c-430a-8cf1-b6295f8b8888.gif)

**Comment by lgants** [comment]
just FYI, I updated the linked reproduction, so this should be ready for review again.

**Comment by afc163** [comment]
Screen Reader works well in Firefox browser but read as `edit text blank` in Chrome.
